### PR TITLE
Fix texture compression

### PR DESCRIPTION
**Pull request by JaimeBea**
Textures were being compressed on import, which is not the default behavior we want. This fixes multiple visual problems with the scene, including:
- Textures having improper colors
- Normal mapping artifacts

It would be a good idea to allow compression / decompression of individual textures, but this will be done separately.

### CODE REVIEW

**Review by lowysole — APPROVED** [review]

File: Project/Source/Resources/ResourceTexture.cpp [hunk 1]
```diff
@@ -39,19 +39,12 @@ void ResourceTexture::Load() {
...
 		LOG("Failed to load image.");
 		return;
 	}
 
-	// Flip image if neccessary
-	ILinfo info;
-	iluGetImageInfo(&info);
-	if (info.Origin == IL_ORIGIN_UPPER_LEFT) {
-		iluFlipImage();
-	}
```
Comment (FrancPS) [inline Project/Source/Resources/ResourceTexture.cpp:53]: Why is this removed¿

Comment (JaimeBea) [reply Project/Source/Resources/ResourceTexture.cpp:53]: Because RAW images don't save orientation information, so I had to change the place where it gets flipped.

**Review by FrancPS — APPROVED** [review]

**Review by telmiuse — APPROVED** [review]
